QueueModel.py

```diff
@@ -48,6 +48,13 @@ def __getstate__(self):
     def __setstate__(self, d):
         self.__dict__.update(d)
 
+    def __getstate__(self):
+        d = dict(self.__dict__)
+        return d
+
+    def __setstate__(self, d):
```

Comment:
You're right - I will fix it today hopefully ; thanks for having accepted the PR see you :D